# exceptions messages
app.err.request=Error while making GET request to {0} to get list of items codes. Response is empty
wb.err.request=Error while making GET request to {0}{1} to get price for item {2}
err.define.price=Failed to define item price for code {0}

user.not.found=User {0} not found
username.already.used=Username {0} already in use
email.already.used=Email {0} already in use

code.already.added=Item code {0} has already been added to the tracking list
code.not.found.userprofile=Item {0} not found in user profile {0}
code.not.found=Item {0} not found

parser.fail.updated.price=Fail while adding updated price from parser. Item with code {0} wasn't found
parser.fail.added.code=Failed to add item code {0}. Please try again later
parser.fail.define.brand=Failed to define item brand
parser.fail.define.name=Failed to define item name
parser.fail.define.price=Failed to define item price

validation.item.brand=Failed to define item brand for code {0}
validation.item.name=Failed to define item name for code {0}
validation.item.price=Failed to define item price for code {0}
